### PR TITLE
README: Add an open source GUI desktop server installer and manager to Resources Section

### PR DESCRIPTION
**Pull request by Jeamee**
## Description
Adding a new open source desktop app that offer install dependencies and browse, install, manage server with ease. Enable non-developer to use MCP Servers without any efforts.

## Motivation and Context
People who want to use MCP server need to:
1. Install node and uv as dependencies
2. Find servers and their install instruction from repos
3. Find and open the deep hidden configuration JSON file
4. Edit the JSON file to insert server config line

This is a too long chain, any step failed will prevent people use MCP server. It's so bad to promote MCP server to more people. And yes, there are some tools like mcp-get tried to solve this problem. But it still need people install dependencies, copy paste and run install cmd in CLI, it need 3 step to install a server, still too long chain.

The new GUI app, enable people install dependencies and install servers by only one click in single place. People can use MCP server without developer experience. The chain been shorten to 1 step. I hope this app will contribute our community grow.

## How Has This Been Tested?
I've test it at Macos 15.1.1 (24B91), include these features:
1. Detect dependencies
2. Install dependencies
3. Browse servers
4. Install servers
5. Install servers with env variable configuration
6. Update env variable configuration
7. Uninstall servers
8. Installed servers run correctly at Claude desktop app

## Breaking Changes
N/A

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Checklist
- [x] I have read the [MCP Protocol Documentation](https://modelcontextprotocol.io)
- [x] My changes follows MCP security best practices
- [ ] I have updated the server's README accordingly
- [x] I have tested this with an LLM client
- [ ] My code follows the repository's style guidelines
- [ ] New and existing tests pass locally
- [ ] I have added appropriate error handling
- [ ] I have documented all environment variables and configuration options

## Additional context
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Additional resources on MCP.
 - **[mcp-get](https://mcp-get.com)** - Command line tool for installing and managing MCP servers by **[Michael Latman](https://github.com/michaellatman)**
 - **[mcp-cli](https://github.com/wong2/mcp-cli)** - A CLI inspector for the Model Context Protocol by **[wong2](https://github.com/wong2)**
 - **[r/mcp](https://www.reddit.com/r/mcp)** – A Reddit community dedicated to MCP by **[Frank Fiegel](https://github.com/punkpeye)**
+- **[MCPHub](https://github.com/Jeamee/MCPHub-Desktop)** – An Open Source GUI Desktop app for discover, installing and managing MCP servers by **[Jeamee](https://github.com/jeamee)**
```
Comment (jspahrsummers) [inline README.md:91]:
```suggestion
- **[MCPHub](https://github.com/Jeamee/MCPHub-Desktop)** – An Open Source GUI Desktop app for discovering, installing and managing MCP servers by **[Jeamee](https://github.com/jeamee)**
```

**Review by jspahrsummers — CHANGES_REQUESTED** [review]
Thanks!

What platform(s) does this support? Can you please mention that as well, and merge from the latest `main`?

**Comment by Jeamee** [comment]
> Thanks!
> 
> What platform(s) does this support? Can you please mention that as well, and merge from the latest `main`?

It supports MacOS && Windows, developed by Tauri 2.0.

**Comment by Jeamee** [comment]
> Thanks!
> 
> What platform(s) does this support? Can you please mention that as well, and merge from the latest `main`?

I've committed the platform mention to the latest `main`, thanks for your suggestion.

**Pull request merged**